boxes/edges.py

```diff
@@ -1461,12 +1461,15 @@ class ChestHingeSettings(Settings):
 
  * pin_height : 2.0 : radius of the disc rotating in the hinge (multiples of thickness)
  * hinge_strength : 1.0 : thickness of the arc holding the pin in place (multiples of thickness)
+ * finger_joints_on_{box,lid} : False : whether to include finger joints on the edge with the box or lid
 """
 
     relative_params = {
         "pin_height" : 2.0,
         "hinge_strength" : 1.0,
         "play" : 0.1,
```

Comment:
These need to be absolute_params. There is really no need to scale the bool with thickness

Reply:
Thanks, fixed